tools/topology/sof-apl-demux-pcm512x.m4

```diff
@@ -151,9 +151,9 @@ DAI_CONFIG(SSP, 5, 0, SSP5-Codec,
 		SSP_TDM(2, 32, 3, 3),
 		SSP_CONFIG_DATA(SSP, 5, 24)))
 
-# 3 HDMI/DP outputs (ID: 1,2,3)
-DAI_CONFIG(HDA, 0, 1, iDisp1)
-DAI_CONFIG(HDA, 1, 2, iDisp2)
-DAI_CONFIG(HDA, 2, 3, iDisp3)
+# 3 HDMI/DP outputs (ID: 3,4,5)
+DAI_CONFIG(HDA, 0, 3, iDisp1)
+DAI_CONFIG(HDA, 1, 4, iDisp2)
+DAI_CONFIG(HDA, 2, 5, iDisp3)
```

Comment:
ok the indices look fine, but I am not sure I understand this line


````
# currently this dai is here as "virtual" capture backend
W_DAI_IN(SSP, 5, SSP5-Codec, s24le, 3, 0)
````

what is the purpose of this line?

What happens if we truly want to use SSP5 for capture?

Reply:
@plbossart I have to test and think that a bit. Frankly I was just now trying to load the demux component for testing my topology work and loading stopped because of wrong idisp indexes. IOW I didn't actually test does the topology work, with this it at least loads.